onnxruntime/core/providers/cpu/ml/label_encoder.h

```diff
@@ -105,5 +106,80 @@ class LabelEncoder_2 final : public OpKernel {
   // ONNX attribute name to load values.
   std::string _value_field_name;
 };
+
+template <typename T>
+std::vector<T> GetAttribute(const OpKernelInfo& info, const std::string& name, const std::string& tensor_name) {
+  std::vector<T> attrs;
+  auto result = info.GetAttrs<T>(name, attrs);
+  if (!result.IsOK()) {
+    ONNX_NAMESPACE::TensorProto attr_tensor_proto;
+    result = info.GetAttr(tensor_name, &attr_tensor_proto);
+    ORT_ENFORCE(result.IsOK(), "LabelEncoder is missing an attribute");
```

Comment:
The error message must include details about the name and the type of the attribute.